src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Color {
+    red,
```

Comment:
enum names should be UpperCase only 
```suggestion
    RED, 
    BLUE...
```